src/App.js

```diff
@@ -1,16 +1,44 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  const [chatEntries, setChatEntries] = useState(chatMessages);
```

Comment:
Awesome set up for your main `chatEntries` state variable and great work setting this at the very top of your component so it's easy for others' to find.